documentation/src/main/asciidoc/stories/assembly_externalizing_spring_session.adoc

```diff
@@ -0,0 +1,10 @@
+[id='externalizing-spring-session']
+:context: spring-session
+
+= Externalizing sessions with Spring Session
```

Comment:
Store session data for Spring applications in {brandname} caches and independently of the container.